### PR TITLE
V2 Move varlink home

### PR DESCRIPTION
**Pull request by jwhonce**
* move cmd/podman/varlink/* to pkg/varlink to support podmanV2 refactor
* update Makefile
* reformatted all impacted code

This does **NOT** include a port to the latest varlink release with Context I/O support.

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
I like this change...

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,7 @@
 __pycache__
 /cmd/podman/varlink/ioprojectatomicpodman.go
 /cmd/podman/varlink/iopodman.go
+/pkg/varlink/iopodman.go
```
Comment (TomSweeneyRedHat) [inline .gitignore]: is the old line, line 22 still needed?

**Comment by TomSweeneyRedHat** [comment]
One question, otherwise LGTM once the tests are happy

File: .gitignore (hunk 1 above)
Comment (jwhonce) [reply .gitignore]: On a future PR, I'll remove both 21 and 22

**Pull request merged**